debug_scripts/get_new_gw_identity/developer_gateway_identity/index.js

```diff
@@ -176,7 +177,9 @@ const run = async() => {
     let identity_obj = {};
 
     let currentSerialNumber = crypto.randomBytes(2).readUInt16BE(0, true);
-    identity_obj.serialNumber = IDGenerator.SerialIDGenerator(program.serialNumberPrefix || 'SOFT', currentSerialNumber, currentSerialNumber + 1);
+    identity_obj.serialNumber = typeof program.serialNumber == 'string' ?
+                                    program.serialNumber :
+                                    IDGenerator.SerialIDGenerator(program.serialNumberPrefix || 'SOFT', currentSerialNumber, currentSerialNumber + 1);
```

Comment:
Can't review this part. No JS experience.

Looks fine?

Reply:
Ya manually verified it on the gateway.